tingbot/cache.py

```diff
@@ -0,0 +1,135 @@
+#image cache
+import time
+import re
+import rfc822
+import calendar
+import requests
+import io
+import os
+from urlparse import urlparse
+
+def get_http_timestamp(dt):
+    return calendar.timegm(rfc822.parsedate(dt))
```

Comment:
Good point. I'll need to look at this tomorrow. Have fixed the other errors and will commit relevant changes